### PR TITLE
ESP-148 Use Flink Json Format in Lookup Source

### PR DESCRIPTION
**Pull request by kristoffSC**
#### Description

 Use Flink Json Format in Lookup Source instead previously created custom json format that was using json-path library.

##### PR Checklist
- [x] Tests added
- [x] [Changelog](CHANGELOG.md) updated

### CODE REVIEW

File: src/main/java/com/getindata/connectors/http/internal/table/lookup/HttpLookupTableSourceFactory.java [hunk 1]
```diff
@@ -164,6 +130,7 @@ private List<String> convertArguments(String arguments) {
         return Arrays.stream(arguments.split(";")).map(String::trim).collect(Collectors.toList());
     }
 
+    // TODO verify this since we are on 1.15 now.
```
Comment (swtwsk) [inline src/main/java/com/getindata/connectors/http/internal/table/lookup/HttpLookupTableSourceFactory.java:133]: What do we want to verify?

**Review by swtwsk — APPROVED** [review]
LGTM, apart from the one vague TODO :) Either add comment here in GitHub or expand the comment in code, and I'll be ok with it :D

File: src/main/java/com/getindata/connectors/http/internal/table/lookup/HttpLookupTableSourceFactory.java (hunk 1 above)
Comment (kristoffSC) [reply src/main/java/com/getindata/connectors/http/internal/table/lookup/HttpLookupTableSourceFactory.java:133]: do I still need to back-port code below from Flink 1.15 :)

Comment (kristoffSC) [reply src/main/java/com/getindata/connectors/http/internal/table/lookup/HttpLookupTableSourceFactory.java:133]: image

**Pull request merged**